documentation/features/scheduling.rst

```diff
@@ -0,0 +1,223 @@
+.. _scheduling:
+
+Scheduling 
+===========
+
+Scheduling is the main value-drive of FlexMeasures. We have two major types of schedulers built-in, for storage devices (usually batteries or hot water storage) and processes (usually in industry).
+
+FlexMeasures computes schedules for energy systems that consist of multiple devices that consume and/or produce electricity.
+We model a device as an asset with a power sensor, and compute schedules only for flexible devices, while taking into account inflexible devices.
+
+.. contents::
+    :local:
+    :depth: 2
+
+
+.. _describing_flexibility:
+
+Describing flexibility
+----------------------
+
+To compute a schedule, FlexMeasures first needs to assess the flexibility state of the system.
+This is described by:
+
+- the ``flex-context`` ― information about the system as a whole, in order to assess the value of activating flexibility.
+- the ``flex model`` ― information about the state and possible actions of the flexible device. We will discuss these per scheduled device type.
+
+This information goes beyond the usual time series recorded by an asset's sensors. It's being sent through the API when triggering schedule computation.
+Some parts of it can be persisted on the asset & sensor model as attributes (that's design work in progress). 
```

Comment:
Here we could reference a GH table we have in one of our Discussions.

Reply:
Actually, [that table](https://github.com/FlexMeasures/flexmeasures/discussions/957) does not talk about support of flex-model or flex-context on assets or sensors. I'd welcome that very much.